app/search_builders/hyrax/works_search_builder.rb

```diff
@@ -1,13 +1,7 @@
 module Hyrax
-  # Finds a list of works. It returns no result if you don't have
-  # access to the requested work.  If the work is suppressed (due to being in a
-  # workflow), then it checks to see if the current_user has any workflow role
-  # on the given work.
-  class WorksSearchBuilder < Hyrax::SearchBuilder
-    include Hyrax::FilterByType
-
-    def only_works?
-      true
-    end
+  # Returns all works, either active or suppressed.
+  # This should only be used by an admin user
+  class WorksSearchBuilder < ActiveWorksSearchBuilder
+    self.default_processor_chain -= [:only_active_works]
```

Comment:
I would expect to see this in the `ActiveWorksSearchBuilder` rather than the `WorksSearchBuilder`. Did those get mixed up, or is it just me?